dpnp/backend/kernels/dpnp_krnl_elemwise.cpp

```diff
@@ -894,55 +653,6 @@ static void func_map_init_elemwise_1arg_1type(func_map_t &fmap)
     fmap[DPNPFuncName::DPNP_FN_ERF_EXT][eft_DBL][eft_DBL] = {
         eft_DBL, (void *)dpnp_erf_c_ext<double>};
 
-    fmap[DPNPFuncName::DPNP_FN_FLATTEN][eft_BLN][eft_BLN] = {
-        eft_BLN, (void *)dpnp_copy_c_default<bool>};
-    fmap[DPNPFuncName::DPNP_FN_FLATTEN][eft_INT][eft_INT] = {
-        eft_INT, (void *)dpnp_copy_c_default<int32_t>};
-    fmap[DPNPFuncName::DPNP_FN_FLATTEN][eft_LNG][eft_LNG] = {
-        eft_LNG, (void *)dpnp_copy_c_default<int64_t>};
-    fmap[DPNPFuncName::DPNP_FN_FLATTEN][eft_FLT][eft_FLT] = {
-        eft_FLT, (void *)dpnp_copy_c_default<float>};
-    fmap[DPNPFuncName::DPNP_FN_FLATTEN][eft_DBL][eft_DBL] = {
-        eft_DBL, (void *)dpnp_copy_c_default<double>};
-    fmap[DPNPFuncName::DPNP_FN_FLATTEN][eft_C128][eft_C128] = {
-        eft_C128, (void *)dpnp_copy_c_default<std::complex<double>>};
-
-    fmap[DPNPFuncName::DPNP_FN_NEGATIVE][eft_INT][eft_INT] = {
-        eft_INT, (void *)dpnp_negative_c_default<int32_t>};
-    fmap[DPNPFuncName::DPNP_FN_NEGATIVE][eft_LNG][eft_LNG] = {
-        eft_LNG, (void *)dpnp_negative_c_default<int64_t>};
-    fmap[DPNPFuncName::DPNP_FN_NEGATIVE][eft_FLT][eft_FLT] = {
-        eft_FLT, (void *)dpnp_negative_c_default<float>};
-    fmap[DPNPFuncName::DPNP_FN_NEGATIVE][eft_DBL][eft_DBL] = {
-        eft_DBL, (void *)dpnp_negative_c_default<double>};
-
-    fmap[DPNPFuncName::DPNP_FN_RECIP][eft_INT][eft_INT] = {
```

Comment:
I believe we have to remove enums which function names which are not used anymore.